### ISSUE TITLE
Add docs for Windows debug info with /Z7

### ISSUE THREAD

**Issue by memsharded** [conan-io/docs#462]
With that option, debug info might be embedded in object files, thus .pdb wouldn't be necessary.

**Comment by lasote** [comment]
Should it be introduced in the visual build helper when debug? Sounds like reasonable.

**Comment by memsharded** [comment]
And why not in the CMake helper too? I am not sure. The final object files would be much bigger by default, and it is not that frequent people stepping into third-parties code. Maybe documenting how to step-in third-party packages while debugging, whatever options they use, could be enough, I don't know...

**Comment by lasote** [comment]
Yes, CMake too. I don't agree that users don't want to step in third-party libraries because is many cases the libraries are their libraries. Could be a thing to opt-in or opt-out in the helpers when debug.

**Comment by memsharded** [comment]
Beware of bintray limits too. PDB and debug info is huge.

**Comment by lasote** [comment]
opt-in then? as a low priority feature, it's a valuable info that the helpers could support nicely...

**Comment by memsharded** [comment]
Yes, I think so.
